txp/Pos/Txp/Logic/Local.hs

```diff
@@ -48,18 +48,16 @@ type TxpLocalWorkMode ctx m =
     , HasLens GenesisUtxo ctx GenesisUtxo
     )
 
--- CHECK: @processTx
--- #processTxDo
+-- | Process transaction. 'TxId' is expected to be the hash of
+-- transaction in 'TxAux'. Separation is supported for optimization
+-- only.
 txProcessTransaction
     :: TxpLocalWorkMode ctx m
     => (TxId, TxAux) -> ExceptT ToilVerFailure m ()
 txProcessTransaction itw@(txId, txAux) = do
     let UnsafeTx {..} = taTx txAux
     tipDB <- GS.getTip
-    epoch <- siEpoch <$>
-        -- TODO: Don't use inaccurate slotting here. If we don't know the
-        -- current slot, we should reject transactions. See CSL-1341.
-        getCurrentSlotInaccurate
+    epoch <- maybe (throwError ToilSlotUnknown) (pure . siEpoch) =<< getCurrentSlot
```

Comment:
>Yes, but that's your subjective point only.

not only me, gromak's too

>PR is a good place to place the concise description of design decisions and features descriptions in IMO.

lol, it's a good place for *decided* decision, I would like to see related discussion, see cons and pros of approaches and maybe participate in them.

you can scroll through unrelated discussion or participate in discussions you want